apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -22,7 +23,20 @@ export const ActionsBanner = ({disabled = false}: {disabled: boolean}) => {
   const wallet = useSelectedWallet()
   const {resetForm} = useSend()
 
-  const onSend = () => {
+  const handleOnBuy = () => {
+    const isMainnetWallet = wallet.networkId === 1
+    const walletAddress = wallet.externalAddresses[0]
+    const banxa = banxaModuleMaker({isProduction: isMainnetWallet, partner: 'emurgo'})
+    const url = banxa.createReferralUrl({
+      coinType: 'ADA',
+      fiatType: 'USD',
+      blockchain: 'ADA',
+      walletAddress,
+    })
+    Linking.openURL(url.toString())
+  }
+
+  const handleOnSend = () => {
```

Comment:
`onPressSend`/`onPressBuy` ? or `handleOnBuy`/`handleOnSend`?

Reply:
and from now on follow the same pattern

Reply:
we usually stick with `onPress` to keep consistency, yet, when as props we use event name `on{Action,Buy,Sell}` in case `onPress` doesn't "fit", we use `handleXXX` as internal functionals, and never as props.